installer/scripts/service_control

```diff
@@ -20,29 +23,75 @@ VAR_DIR=/var/opt/microsoft/omsagent
 ETC_DIR=/etc/opt/microsoft/omsagent
 BIN_DIR=/opt/microsoft/omsagent/bin
 
-WORKSPACE_REGEX='^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$'
+REGEX_UUID='[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}'
+WORKSPACE_REGEX="^$REGEX_UUID$"
+VALIDATION_SUPPLEMENT_MESSAGE=
+VALIDATION_SEVERITY=WARNING
 
 # Space seperated list of non oms workspaces
 NON_OMS_WS="scom"
 
-setup_variables()
-{
+# Begin Lower Level Utilities
+
+validate_file_has_line_pattern() {
+	local _fSpec=$1
+	local _pattA="$2"
+	local _boolOnly=$3
+	cat $_fSpec | grep -Eq "^$_pattA$"
+	if [ $? -eq 0 ]; then
+		return 0
+	fi
+	if [ -z "$_boolOnly" ]; then
+		echo "$VALIDATION_SEVERITY:  $_fSpec missing required pattern '$_pattA'."
+        echo $VALIDATION_SUPPLEMENT_MESSAGE
```

Comment:
Fix indentation on this line; it's not aligned with the line above it.

Reply:
I'll go through the entire file and fix these.  I realized it's from bringing in the routines from my unit-tested areas where I didn't have the python tabbing.